### PR TITLE
Filter mesure

### PR DESCRIPTION
**Pull request by PierreGoe**

### CODE REVIEW

**Review by Mariion-62 — DISMISSED** [review]

File: src/Components/Filter/style.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+const SRadio = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  .radLabel {
+    width: 128px;
+    display: flex;
+    align-items: center;
+    background-color: white;
+
```
Comment (Hacene010) [inline src/Components/Filter/style.jsx]: empty line !

File: src/Components/Filter/style.jsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    border-radius: 100px;
+    padding: 14px 16px;
+    margin: 10px 0;
+
```
Comment (Hacene010) [inline src/Components/Filter/style.jsx]: empty line !

File: src/Components/Filter/style.jsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    height: 1px;
+    opacity: 0;
+    z-index: -1;
+  }
+
+  .radDesign {
+    width: 22px;
+    height: 22px;
+    border-radius: 100px;
+
```
Comment (Hacene010) [inline src/Components/Filter/style.jsx]: empty line !

File: src/Components/Filter/style.jsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      to right bottom,
+      hsl(154, 97%, 62%),
+      hsl(225, 97%, 62%)
+    );
+    position: relative;
+  }
+
+  .radDesign::before {
+    content: '';
+
```
Comment (Hacene010) [inline src/Components/Filter/style.jsx]: empty line !

File: src/Components/Filter/style.jsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    display: inline-block;
+    width: inherit;
+    height: inherit;
+    border-radius: inherit;
+
```
Comment (Hacene010) [inline src/Components/Filter/style.jsx]: empty line !

File: src/Components/Filter/style.jsx [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  }
+
+  .radText {
+    color: hsl(0, 0%, 60%);
+    margin-left: 14px;
+    letter-spacing: 3px;
+    text-transform: uppercase;
+    font-size: 18px;
+    font-weight: 900;
+
```
Comment (Hacene010) [inline src/Components/Filter/style.jsx]: empty line !

File: src/Components/MarkerIndoor/index.jsx [hunk 7]
```diff
@@ -1,19 +1,42 @@
 import axios from 'axios';
-import { useState, useEffect } from 'react';
+import { useState, useEffect, useContext } from 'react';
 import L from 'leaflet';
```
Comment (Hacene010) [inline src/Components/MarkerIndoor/index.jsx:3]: Think about the next dev who will work on this project :-P (What is L for ?)

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some empty lines to remove :-)

**Review by Mariion-62 — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**